src/convnets/alexnet.jl

```diff
@@ -42,7 +42,7 @@ Create a `AlexNet`.
   - `nclasses`: the number of output classes
 
 !!! warning
-    
```

Comment:
There are a number of files with unrelated changes like this that make the review difficult. Although removing trailing whitespace is always good, it's better left as its own PR. For this one, can you revert the changes to unrelated files so that we can focus on directly relevant ones?